### PR TITLE
Tdh 144

### PR DESCRIPTION
**Pull request by JocelynPi**

### CODE REVIEW

File: dashboardv2/public/js/views/entity/CreateEntityLayoutView.js [hunk 1]
```diff
@@ -869,8 +869,18 @@ define(['require',
...
+                        } else {
+                            if (_.isObject(relationshipType) && !_.isArray(relationshipType)) {
+                                relationshipType = [relationshipType];
+                            }
+                            _.each(relationshipType, function(obj) {
+                                if (_.isObject(obj) && obj.guid ) {
+                                    obj['id'] = obj.guid
+                                    select2Options.push(obj);
+                                    selectedValue.push(obj.guid);
+                                }
```
Comment (JocelynPi) [inline dashboardv2/public/js/views/entity/CreateEntityLayoutView.js:882]: @chongyk-heetch is this related to the rebase that you did ?

Comment (Axelcouty) [reply dashboardv2/public/js/views/entity/CreateEntityLayoutView.js:882]: Oh it's not yours @JocelynPi ?

Comment (JocelynPi) [reply dashboardv2/public/js/views/entity/CreateEntityLayoutView.js:882]: Not at all :'(

Comment (chongyk-heetch) [reply dashboardv2/public/js/views/entity/CreateEntityLayoutView.js:882]: i think it comes from the rebase yes
